### PR TITLE
Максименко Александр, 2538

### PR DESCRIPTION
**Pull request by FluffyBear**

### CODE REVIEW

File: app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+ * Created by thevery on 11/09/14.
+ */
 class WhirlView extends SurfaceView implements Runnable {
     int [][] field = null;
-    int width = 0;
-    int height = 0;
-    int scale = 4;
+    Bitmap bitmap;
+    Rect src, dst;
+    int width = 0, height = 0;
```
Comment (svasilinets) [inline app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java:20]: Please never define fields on the same line

File: app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java [hunk 2]
```diff
@@ -44,55 +44,54 @@ public void pause() {
     public void run() {
         while (running) {
             if (holder.getSurface().isValid()) {
-                long startTime = System.nanoTime();
                 Canvas canvas = holder.lockCanvas();
                 updateField();
                 onDraw(canvas);
                 holder.unlockCanvasAndPost(canvas);
-                long finishTime = System.nanoTime();
-                Log.i("TIME", "Circle: " + (finishTime - startTime) / 1000000);
```
Comment (svasilinets) [inline app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java:53]: Please return time logging.

**Comment by svasilinets** [comment]
Looks ok overall, but please add back logging of the time to measure a fps of your application.
No penalty will be applied for that commit after the deadline.

**Comment by FluffyBear** [comment]
ok

**Comment by FluffyBear** [comment]
I did it

**Comment by svasilinets** [comment]
~ 12 fps. ok.

**Pull request closed without merging**